### DISCUSSION TITLE
请问grok开源的代码里，是不是没有它训练的资料？

### DISCUSSION THREAD

**Discussion by jimys** [xai-org/grok-1#247] [General]
请问grok开源的代码里，是不是没有它训练的资料？也就是大量互联网搜集的文字数据，比如大量说推特的推文，或者微博知乎各种百科和整个网络的文字，

开源里面好像没有这些资料？

那是否意味着即使我有了grok源代码，其实还得自己去搜集文字资料，喂给它去训练模型？

**Comment by YeexiaoZheng** [comment]
权重也有的 磁力链 or hugging face里就有

**Comment by xiciliu** [comment]
里面没有的，需要另外下载，在readme中有写。
https://www.yuque.com/u21594513/pwky0v/fe0lcl5m7oqwfy4l?singleDoc# 《grok-1安装指南》
